src/components/DocumentRenderer/DocumentDetailsSheet.tsx

```diff
@@ -254,11 +284,17 @@ export const DocumentDetailsSheet: FunctionComponent<DocumentDetailsSheet> = ({
 
   let priorityContent: ReactNode = null;
   if (haveChecksFinished) {
+    console.log(hasExpired);
     let children: ReactNode = null;
-    if (isDocumentValid) {
+    if (isDocumentValid && !hasExpired) {
       children = (
-        <View style={styles.qrCodeWrapper}>
-          <QrCode qrCode={qrCode} qrCodeLoading={qrCodeLoading} />
+        <View>
+          <View style={styles.qrCodeWrapper}>
+            <QrCode qrCode={qrCode} qrCodeLoading={qrCodeLoading} />
+            {ttlDescription ? (
+              <Text style={styles.ttlText}>Expires in {ttlDescription}</Text>
+            ) : null}
```

Comment:
This approach couples the idea of `ttlDescription` with the way it's presented. It'll be better if we store the `ttl` in state, then let render figure out how to present the TTL in a human readable form.